checkstyle.xml

```diff
@@ -3,7 +3,7 @@
 <module name="Checker">
     <!-- Headers -->
     <module name="RegexpHeader">
-        <property name="headerFile" value="license.header" />
+        <!--<property name="headerFile" value="license.header" />-->
```

Comment:
not sure why the licence.header file won;t pass the regex on windows.

Reply:
Yes, when I was playing with xunit-plugin, I was getting this error

Reply:
yeah I won't fight this one